packages/client/bin/cli.ts

```diff
@@ -80,10 +80,15 @@ const args = require('yargs')
       boolean: true,
       default: Config.RPC_DEFAULT,
     },
-    rpcport: {
+    rpcHttpPort: {
       describe: 'HTTP-RPC server listening port',
       number: true,
-      default: Config.RPCPORT_DEFAULT,
+      default: Config.RPCHTTPPORT_DEFAULT,
+    },
+    rpcWssPort: {
+      describe: 'WSS-RPC server listening port',
+      number: true,
+      default: Config.RPCWSSPORT_DEFAULT,
```

Comment:
I know this is still WIP, but just to mention: we likely also want to have the option to activate/deactivate separtely here? So with `--rpcWss` I guess?

Side question: is this `Wss` the `Wss` from `Web Socket Secure` as mentioned in Wikipedia? 🤔

> The WebSocket protocol specification defines ws (WebSocket) and wss (WebSocket Secure)

Reply:
> I know this is still WIP, but just to mention: we likely also want to have the option to activate/deactivate separtely here? So with `--rpcWss` I guess?
> 
Ah, good catch.  I'll update that in my next commit.
> Side question: is this `Wss` the `Wss` from `Web Socket Secure` as mentioned in Wikipedia? thinking
> 
> > The WebSocket protocol specification defines ws (WebSocket) and wss (WebSocket Secure)

I guess I was thinking WebSocket secure but this truly will be `ws://...`.  Will update and have better names args.

Reply:
for default-on values it would be cool to also work with them being passed as `false` to turn them off. I know WS probably won't be on by default, but if it was, then e.g. to disable them with `--rpcWss=false` - it would be nice to support this across the other default-on values in general so there is an expected default way to disable things. Just a thought!